pages/apps/[app].tsx

```diff
@@ -5,6 +5,8 @@ import { GetApp } from "../../components/GetApp";
 import Layout from "../../components/Layout";
 import { AppInfo } from "../../interfaces/app";
 
+builder.init("c33bcd23c29e45789677ba9aaaa7ce1d");
```

Comment:
I _think_ this isn't needed bc it's in `pages/_app.tsx` which runs for all pages

Reply:
I was getting this error:
![image](https://user-images.githubusercontent.com/581205/105433118-e0c0e400-5c0d-11eb-9f94-be161b882254.png)